TestResultSummaryService/routes/getBenchmarkVariants.js

```diff
@@ -0,0 +1,5 @@
+const BenchmarkMetric = require( '../parsers/BenchmarkMetric');
+const BenchmarkRouter = require( '../parsers/BenchmarkMetricRouter');
+module.exports = async ( req, res ) => {
+    res.json({'benchmarkMetricRouter':BenchmarkRouter,'benchmarkMetric':BenchmarkMetric});
```

Comment:
Indent!

Reply:
Let's just keep it the same name instead of `benchmarkMetricRouter` if possible.